tutorials/ExploringGroversAlgorithm/ExploringGroversAlgorithmTutorial.ipynb

```diff
@@ -363,7 +371,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "timeout"
```

Comment:
I would mark the rest of the cells in this notebook as "invalid_code" as well - technically it's valid but it's clearly not the complete code for this task, so we shouldn't be validating it as such.

Reply:
Changed the reason from timeout to invalid_code for Exercise 4 and Exercise 5 in ExploringGroversAlgorithmTutorial.